lib/flipper/views/feature.erb

```diff
@@ -0,0 +1,218 @@
+<% if params.key?("error") %>
+  <div class="alert alert-danger">
+    <%= params["error"] %>
+  </div>
+<% end %>
+
+<div class="vads-l-col--9">
+  <div class="vads-l-row">
+    <div class="vads-l-col">
+      <h1 class="vads-u-font-size--xl">
+        <%= @feature.key %>
+      </h1>
+
+      <p class="vads-u-font-size--sm">
+        <%= yaml_features.dig(@feature.key,'description') %>
+      </p>
+
+      <div class="feature">
+        <p>
+          <% if @feature.on? %>
+            This feature is <strong>enabled for everyone</strong>.
+          <% elsif @feature.off? %>
+            This feature is <strong>disabled</strong>.
+          <% else %>
+            This feature is <strong>enabled with certain conditions</strong>.
+          <% end %>
+        </p>
+
+        <% if @feature.boolean_value %>
+        <p>Conditional toggles are currently disabled. If you want to conditionally enable this feature, you must first disable the feature for everyone.</p>
```

Comment:
This observation came from testing out the toggles; it looks like the main toggle button is the "boolean gate", and it pretty much overrides any conditional toggles. But it doesn't look like there are currently any methods to only disable the boolean gate, so we'll need to disable everything before setting the conditional toggles again in order to activate their effects.